docs/examples/iq_plot_listen.py

```diff
@@ -0,0 +1,360 @@
+# Copyright 2020 SuperDARN Canada, University of Saskatchewan
+# Author: Kevin Krieger & Ashton Reimer
+# DISCLAIMER: This code is licensed under GPL v3.0 which can be located in LICENSE.md in the root directory
+"""
+Utilities for plotting and listening to DMAP IQ files
+
+Methods
+-------
+iq_to_wav: Create a WAV file out of all records in the input iqdat file.
+Optionally use the I and Q signals to make a stereo WAV file (note: doubles the file size).
+The --play flag will call this if executing the script directly. The --stereo flag enables stereo.
+
+plot_iq: Create a plot of I and Q vs sample number for every record in the input iqdat file.
+The --plot flag will call this if executing the script directly.
+NOTE: This method takes a long time.
+
+get_stats_from_record: Print out some useful stats about the input iqdat file. The --debug
+flag will call this if executing the script directly.
+
+open_compressed_file: General function that takes an input filename, and determines how to open
+it by reading the first few 'magic bytes' from the file. Returns a file stream acceptable to use
+with: pydarn.SDarnRead(filestream, True). The input iqdat file is opened with this method
+if executing this script directly.
+
+References
+----------
+https://www.garykessler.net/library/file_sigs.html
+https://superdarn.github.io/rst/superdarn/src.doc/rfc/0027.html
+
+"""
+import pydarn
+import binascii
+import numpy as np
+import datetime as dt
+import soundfile
+import matplotlib.pyplot as plt
+import argparse as ap
+import os
+
+
```

Comment:
The wav file generated by --listen are not two hours in length whereas the --plot command  generates iq plots for two hours